modules/custom/d_p_side_by_side/templates/paragraph--d-p-side-by-side.html.twig

```diff
@@ -46,12 +46,16 @@
   view_mode ? 'paragraph--view-mode--' ~ view_mode|clean_class,
 ] %}
 
+{% if not has_header %}
+  {% set classes = classes|merge(['without-header']) %}
```

Comment:
Maybe this: `!has_header ? 'without-header'` would work like `view_mode` above? To avoid adding `if`